src/main/java/org/folio/search/service/reindex/jdbc/ContributorRepository.java

```diff
@@ -71,15 +74,68 @@ public class ContributorRepository extends UploadRangeRepository implements Inst
         c.id;
     """;
 
+  private static final String SELECT_BY_UPDATED_QUERY = """
+    WITH cte AS (SELECT id,
+                        name,
+                        name_type_id,
+                        authority_id,
+                        last_updated_date
+                 FROM %1$s.contributor
+                 WHERE last_updated_date > ?)
```

Comment:
Would be better to have a limit on those queries, and better be configurable with default value of 500